perfkitbenchmarker/providers/ibmcloud/ibmcloud_disk.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module containing classes related to IBM Cloud disks """
+
+import json
+import logging
+import time
+import re
+
+from absl import flags
+from perfkitbenchmarker import disk
+from perfkitbenchmarker import errors
+from perfkitbenchmarker import vm_util
+
+from perfkitbenchmarker.providers.ibmcloud import ibm_api as ibm
+
+FLAGS = flags.FLAGS
+
+ATTACHED = 'attached'
+AVAILABLE = 'available'
+DEFAULT_DISK_SIZE = 1000
+
+
+class IbmCloudDisk(disk.BaseDisk):
+  """Object representing a IBM Cloud Disk"""
+
+  def __init__(self, disk_spec, name, zone):
```

Comment:
For new code can you pytype it?  In this case

def __init__(self, disk_spec: disk.DiskSpec, name: str, zone:str)

I realize it looks like a pain to do but it really helps when looking at it later

Reply:
agree, thanks for the tip.